pkg/cmd/cli/install/install.go

```diff
@@ -110,6 +111,7 @@ func (o *Options) BindFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.RestoreOnly, "restore-only", o.RestoreOnly, "Run the server in restore-only mode. Optional.")
 	flags.BoolVar(&o.DryRun, "dry-run", o.DryRun, "Generate resources, but don't send them to the cluster. Use with -o. Optional.")
 	flags.BoolVar(&o.UseNodeAgent, "use-node-agent", o.UseNodeAgent, "Create Velero node-agent daemonset. Optional. Velero node-agent hosts Velero modules that need to run in one or more nodes(i.e. Restic, Kopia).")
+	flags.BoolVar(&o.PrivilegedAgent, "privileged-agent", o.PrivilegedAgent, "Use privileged mode for the node agent. Optional. Required to backup block devices.")
```

Comment:
The existing examples of privileged mode are a special case for OpenShift.
If the block mode needs the node agent to run in privileged mode, I think it's reasonable to add a new parameter, but the name is not accurate enough. It can make the user think this parameter will make the Velero server run the privileged mode instead of the node agent.

Reply:
I modified the 2 documents to use the new flag, so this should be resolved.